requirements.txt

```diff
@@ -1,4 +1,4 @@
-jsonschema
+jsonschema==3.2.0 ; python_version == '3.5'
```

Comment:
Should we make this <= 3.5, that way the Python 2.7 jobs will always install the correct version?